tmt/checks/avc.py

```diff
@@ -35,11 +35,7 @@ def _save_report(
             datetime.datetime.now(datetime.timezone.utc))
 
         assert invocation.phase.step.workdir is not None  # narrow type
-
-        path = invocation.data_path(
-            filename=TEST_POST_AVC_FILENAME.format(event=event.value),
-            create=True,
-            full=True)
+        path = invocation.check_files_path / TEST_POST_AVC_FILENAME.format(event=event.value)
```

Comment:
Now when we have our own little directory, I'd drop the `tft-` prefix from filenames created by the AVC and dmesg checks :)